src/components/DashboardTable/DashboardTable.tsx

```diff
@@ -8,25 +9,22 @@ import TableRow from '@mui/material/TableRow';
 import Paper from '@mui/material/Paper';
 import { EditButton } from './DashboardTable.styled';
 
-interface DashboardTableProps {
-  isActual: boolean;
+interface Vacation {
```

Comment:
in App.tsx there is exactly the same interface. Please try to avoid interface duplications.

Reply:
done